pkg/kubelet/cm/topologymanager/topology_manager.go

```diff
@@ -233,43 +251,86 @@ func (m *manager) Admit(attrs *lifecycle.PodAdmitAttributes) lifecycle.PodAdmitR
 	klog.Infof("[topologymanager] Topology Admit Handler")
 	pod := attrs.Pod
 
-	for _, container := range append(pod.Spec.InitContainers, pod.Spec.Containers...) {
-		if m.policy.Name() == PolicyNone {
+	// Exception - Policy : none
+	if m.policy.Name() == PolicyNone {
+		for _, container := range append(pod.Spec.InitContainers, pod.Spec.Containers...) {
 			err := m.allocateAlignedResources(pod, &container)
 			if err != nil {
-				return lifecycle.PodAdmitResult{
-					Message: fmt.Sprintf("Allocate failed due to %v, which is unexpected", err),
-					Reason:  "UnexpectedAdmissionError",
-					Admit:   false,
-				}
+				return podAdmitError(unexpectedAdmissionError, err)
 			}
-			continue
 		}
+		return lifecycle.PodAdmitResult{Admit: true}
+	}
+	klog.Infof("[topologymanager] Topology Scope : %v", m.scope)
```

Comment:
This log is redundant given the log line above:
https://github.com/kubernetes/kubernetes/pull/92967/files#diff-a19f204f56e91b0871491b88c0e0b139R139
```
klog.Infof("[topologymanager] Creating topology manager with %s policy per %s scope", topologyPolicyName, topologyScopeName)
```